redhat-developer/try-in-dev-spaces-browser-extension
# Introduce react

With this PR, the injected button is created with React.

To test this PR, 

1. Checkout this PR, run `yarn && yarn test`, and ensure all tests pass
2. Run `yarn build` and sideload the generated extension generated in `dist/chromium`
3. Verify that the button is injected and is working as before
